spec/parsers/maven_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'pry'
```

Comment:
I'd say just put this in `spec_helper.rb` and make things better for all of us :)

Reply:
I have a snippet that adds a `require "pry"; binding.pry` when I type pry<tab> just in case haha. That sounds like a good idea tho ☝️

Reply:
I keep forgetting to remove these things before I push. Should I just add it to `spec_helper`?